tcms/testcases/static/testcases/js/search.js

```diff
@@ -124,3 +135,15 @@ $(document).ready(function() {
 
     $('.selectpicker').selectpicker();
 });
+
+function initDateTimePicker(selector) {
+    $(selector).datetimepicker({
+        locale: '{{ LANGUAGE_CODE }}',
```

Comment:
There is a reason this function is in the HTML template. Variable substitution will not work in JavaScript b/c this doesn't go through Django's templating engine.

Reply:
You can move this into a separate HTML template and include it where necessary. Also moving this function into utils.js